TACC/Frontera-Portal
# bug/FP-828 - Prevent write operations on protected files

## Overview: ##

Prevent move, trash, rename operations on protected files .bash_history, .bash_profile, .bashrc

## Related Jira tickets: ##

* [FP-828](https://jira.tacc.utexas.edu/browse/FP-828)

## Summary of Changes: ##

Add extra condition that prevents write operations on protected files

## Testing Steps: ##
1. Select one of those files
2. Make sure rename, move and trash are not enabled in the toolbar

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/105766806-ce9db900-5f1f-11eb-8704-0b847f133148.png)
